jekyll/_cci2/ios-migrating-from-1-2.md

```diff
@@ -0,0 +1,406 @@
+---
+layout: classic-docs
+title: "Migrating your iOS project from 1.0 to 2.0"
+short-title: "Migrating your iOS project from 1.0 to 2.0"
+description: "How to migrate your iOS project from CircleCI 1.0 to 2.0"
+categories: [migration]
+order: 16
```

Comment:
Need help with this—what should the value be for the doc to show up in the `Migration` section  after the general Migration doc?